independent-projects/resteasy-reactive/common/runtime/src/main/java/org/jboss/resteasy/reactive/common/jaxrs/StatusTypeImpl.java

```diff
@@ -31,6 +41,20 @@ public String getReasonPhrase() {
         return reasonPhrase;
     }
 
+    @Override
+    public boolean equals(Object other) {
+        if (!(other instanceof StatusType)) {
+            return false;
+        }
+        return this.status == ((StatusType) other).getStatusCode()
+                && this.reasonPhrase.equals(((StatusType) other).getReasonPhrase());
```

Comment:
Do we really care about the phrase? Wouldn't the code be enough?

Reply:
@geoand  I think it was failing without it, but I'll check; I'm surprised though that the OIDC tests failed. I don't think it is related, I wonder if some instability has been introduced with KC 19.0.1...

Reply:
@geoand I've checked it, without comparing the reason phrases it fails, as it is not symmetric without this check.

As far as the impact on `keycloak-admin-client-reactive` is concerned, it is quite minimal, but IMHO it makes sense to fix it

Reply:
Run failing OIDC tests many times, looks like it was something transient, in either case a no token case request failed to succeed in getting to the endpoint...